From: Jo-Philipp Wich <jo@mein.io>
Date: Sun, 29 May 2016 15:40:51 +0200
Subject: download.mk: use HTTPS for git.lede-project.org

Some Git versions have issues following the HTTP->HTTPS redirect and since
the keyring package is fetched from this host, switching to HTTPS is a
sensible choice anyway.

Signed-off-by: Jo-Philipp Wich <jo@mein.io>

diff --git a/include/download.mk b/include/download.mk
index 430ef3a92de1d0d98593d3d676370b20e05f8fd1..d8074855f87ef3f269f99050911f2dd167812226 100644
--- a/include/download.mk
+++ b/include/download.mk
@@ -6,7 +6,7 @@
 #
 
 OPENWRT_GIT = http://git.openwrt.org
-LEDE_GIT = http://git.lede-project.org
+LEDE_GIT = https://git.lede-project.org
 
 DOWNLOAD_RDEP=$(STAMP_PREPARED) $(HOST_STAMP_PREPARED)
 
